packages/client/src/utils/localStorageUtils.ts

```diff
@@ -0,0 +1,9 @@
+const USER_NAME = 'userName';
+
+export function persistUserName(userName: string) {
+  return localStorage.setItem(USER_NAME, userName);
+}
+
+export function getPersistedUserName() {
```

Comment:
```suggestion
export function getPersistentUserName() {
```
Or `getStoredUsername`